app/views/devise/sessions/new.html.erb

```
@@ -1,6 +1,10 @@
 <section class="form container">
   <h2>Welcome back!</h2>
   <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |form| %>
+    <div class="notice-text">
+      <p>This is a closed platform. If you have signed up, you must wait for an admin to approve your account before
+        you can login.</p>
+    </div>
```

Do you think it would be a better experience to have this as a custom error page?
The HTTP 401 error code is specifically "Unauthorised" we could throw this error and redirect to an 401 error page.

---

Removed this text and added as flash message